cmd/expiration-mailer/main_test.go

```diff
@@ -77,9 +77,9 @@ const emailARaw = "rolandshoemaker@gmail.com"
 const emailBRaw = "test@gmail.com"
 
 var (
-	emailA, _ = core.ParseAcmeURL("mailto:" + emailARaw)
-	emailB, _ = core.ParseAcmeURL("mailto:" + emailBRaw)
-	jsonKeyA  = []byte(`{
+	emailA   = "mailto:" + emailARaw
+	emailB   = "mailto:" + emailBRaw
```

Comment:
nit: looks like the `, _` was replaced by whitespace. Can you trim it down to `emailA = "mailto:" + emailARaw`, etc.?